.github/workflows/python-publish.yml

```diff
@@ -14,10 +14,10 @@ jobs:
   deploy:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v3
-    - name: Build and publish to pypi
-      uses: JRubics/poetry-publish@v2.0
-      with:
-        pypi_token: ${{ secrets.PYPI_API_TOKEN }}
-        # python_version: "3.10"
-        # poetry_version: "==1.8" # can lock versions if we want
+    - uses: actions/checkout@v4
+    - uses: astral-sh/setup-uv@v6
+    - run: uv build
+    - name: Publish to pypi
+      env:
+        PYPI_API_TOKEN: ${{ secrets.PYPI_API_TOKEN }}
+      run: uv publish --token "$PYPI_API_TOKEN"
```

Comment:
I've never published anything using `uv`, but looking at this and comparing with the [docs](https://docs.astral.sh/uv/reference/cli/#uv-publish): Do we not need to specify the `index` to publish to?

Reply:
Ah you're right! Thanks